test/Syntax/round_trip_parse_gen.swift

```diff
@@ -85,6 +85,16 @@ typealias I = (A & B<C>) -> C & D
 @objc private typealias T<a,b> = Int
 @objc private typealias T<a,b>
 
+class Foo {
+  let bar: Int
+}
+
+class Bar: Foo {
+  var foo: Int = 42
+}
+
+class C<A, B> where A: Foo, B == Bar {}
```

Comment:
could we also add a test using decl attributes? like:
```
@available(*, unavailable)
private class C {}
```

Reply:
Yup. Done.